front/lib/api/assistant/configuration.ts

```diff
@@ -1806,7 +1794,6 @@ export async function updateAgentPermissions(
 }
 
 export function getAgentPermissions(
-  auth: Authenticator,
```

Comment:
NIT: I would remove this function since used only once and not easily reusable elsewhere. WDYT?